authenticator/interactors/implementation/user_manager.go

```diff
@@ -19,7 +20,12 @@ func NewUserManager(auth interactors.Authenticator, repo user.Repository) intera
 }
 
 func (um userManager) Create(user structures.User) (structures.User, error) {
+	loggerFields := logrus.Fields{"email": user.Email, "meth": "userManager:Create"}
+	contextLogger := tools.Log().WithFields(loggerFields)
+
 	if user, err := um.generatePasswordIfEmpty(&user); err != nil {
+		contextLogger.WithError(err).Debug("error creating user with empty password")
```

Comment:
This should be of `error` level. We only see Debug level in our local machines, that means that if this problem happens in a production environment we won't be able to see the log in the log files (because production only logs info, error, warn and fatal, not debug).